tests/reporting/test_bond_redemption.py

```diff
@@ -69,12 +69,12 @@ def test_failed_crowdsourcer(finalize, localFixture, universe, market, cash, rep
 
     if finalize:
         # Fast forward time until the fee window is over and we can redeem to recieve the REP back
```

Comment:
Looks like there are still some references to "fee window" as 2 words. Recommend doing a find/replace on those as well.